src/Features/Core/Portable/CodeRefactorings/CodeRefactoringService.cs

```diff
@@ -83,8 +94,45 @@ private ConcatImmutableArray<CodeRefactoringProvider> GetProviders(Document docu
 
         public async Task<bool> HasRefactoringsAsync(
             Document document,
-            TextSpan state,
+            TextSpan textSpan,
+            TextSpan? pastedTextSpan,
             CancellationToken cancellationToken)
+        {
+            var client = await RemoteHostClient.TryGetClientAsync(document.Project, cancellationToken).ConfigureAwait(false);
+            if (client is not null)
+            {
+                var result = await client.TryInvokeAsync<IRemoteDiagnosticAnalyzerService, bool>(
+                    document.Project.Solution,
+                    invocation: (service, solutionInfo, cancellationToken) => service.HasRefactoringsAsync(solutionInfo, document.Id, textSpan, pastedTextSpan, cancellationToken),
+                    cancellationToken).ConfigureAwait(false);
+
+                if (result.HasValue)
+                {
+                    return result.Value;
+                }
```

Comment:
i believe we're no longer following this pattern.  if the operation fails, we do not try to fallback to local processing.  instead, we jsut bail out (As we will also have had a yellowbar here).  note we may not be 100% consistent here, but i believe that Tomas' strong position on what this ll needs to be.